site-command.php

```diff
@@ -13,4 +13,11 @@
 	require_once $autoload;
 }
 
+// Load utility functions.
+require_once 'src/helper/site-utils.php';
```

Comment:
add file using abs path.
Use __DIR__ . '/src/helper/site-utills.php';
Do same for other places also.